cmd/ocsp-responder/main.go

```diff
@@ -234,6 +234,20 @@ as generated by Boulder's single-ocsp command.
 	<-forever
 }
 
+// ocspMux implements the Handler method exposed by http.ServeMux which we use in the WFE
+// but removes all of the internal path cleaning stuff it does as it can break OCSP requests.
```

Comment:
Instead of "removes" we should say "doesn't implement", and instead of "path cleaning stuff," we should just say "path cleaning." Also I'd leave out "as it can break OCSP requests" because you explain that better in the next sentence.